packages/core/integration-tests/test/integration/scope-hoisting/es6/async-named-import-ns-reexport/reexports.js

```diff
@@ -0,0 +1 @@
+export * as ns from './ns';
```

Comment:
Should we also test:

```
import * as ns from './ns';

export {ns}
```

Reply:
Yes, this wouldn't currently trigger this bug but the tests should test all use cases, not our implementation details.